System/Process.hs

```diff
@@ -200,6 +200,15 @@ always the desired behavior. In cases where you would like to leave the
 instead. All created @Handle@s are initially in text mode; if you need them
 to be in binary mode then use 'hSetBinaryMode'.
 
+'use_process_jobs' can be set in CreateProcess since 1.5.0.0 in order to create
+an I/O completion port to monitor a process tree's progress on Windows.
+The function returns two new handles (inside @/ph/@):
+
+  * an I/O Completion Port handle on which events will be signaled.
```

Comment:
This is no longer true. We don't probe for events anymore so this handle is no longer given. Comment is outdated.

Reply:
Is the following wrong then? It is still talking about events.

https://hackage.haskell.org/package/process-1.6.10.0/docs/System-Process-Internals.html#t:ProcessHandle__

> On Windows a second wait method can be used to block for event completion. This requires two handles. A process job handle and a events handle to monitor.

Also, should this comment be attached to `ProcessHandle` rather than `ProcessHandle__`?  The latter is internal.

Reply:
I did the `ProcessHandle`/`ProcessHandle__` Haddock switch at https://github.com/haskell/process/pull/194